cardano-cli/src/Cardano/CLI/Commands/Governance.hs

```diff
@@ -61,6 +64,61 @@ data GovernanceCmdError
   | ShelleyGovernanceCmdGenesisDelegationNotSupportedInConway
   deriving Show
 
+instance Error GovernanceCmdError where
+  displayError = \case
+    StakeCredGovCmdError stakeAddressCmdError ->
+      "Stake credential error: " <> toS (renderShelleyStakeAddressCmdError stakeAddressCmdError)
+    VotingCredentialDecodeGovCmdEror decoderError ->
+      "Could not decode voting credential: " <> renderDecoderError decoderError
+    WriteFileError fileError ->
+      "Cannot write file: " <> displayError fileError
+    ReadFileError fileError ->
+      "Cannot read file: " <> displayError fileError
+    ExpectedStakeKeyCredentialGovCmdError ->
+      "Unsupported stake credential. Expected stake key credential."
+    NonUtf8EncodedConstitution unicodeException ->
+      "Constitution encoded in a format different than UTF-8: " <> show unicodeException
+    GovernanceCmdTextEnvReadError fileError ->
+      "Cannot read text envelope: " <> displayError fileError
+    GovernanceCmdCddlError cddlError ->
+      "Reading transaction CDDL file error: " <> displayError cddlError
+    GovernanceCmdKeyReadError fileError ->
+      "Cannot read key: " <> displayError fileError
+    GovernanceCmdCostModelReadError fileError ->
+      "Cannot read cost model: " <> displayError fileError
+    GovernanceCmdTextEnvWriteError fileError ->
+      "Cannot write text envelope: " <> displayError fileError
+    GovernanceCmdEmptyUpdateProposalError ->
+      "Empty update proposals are not allowed."
+    GovernanceCmdMIRCertificateKeyRewardMistmach fp nStakeVerKeys nRewards ->
+      "Error creating the MIR certificate at: " <> fp
+      <> " The number of staking keys: " <> show nStakeVerKeys
+      <> " and the number of reward amounts: " <> show nRewards
+      <> " are not equivalent."
+    GovernanceCmdCostModelsJsonDecodeErr fp msg ->
+      "Error decoding cost model: " <> toS msg <> " at: " <> fp
+    GovernanceCmdEmptyCostModel fp ->
+      "The decoded cost model was empty at: " <> fp
+    GovernanceCmdUnexpectedKeyType expectedTypes ->
+      "Unexpected poll key type; expected one of: "
+      <> intercalate ", " (show <$> expectedTypes)
+    GovernanceCmdPollOutOfBoundAnswer maxIdx ->
+      "Poll answer out of bounds. Choices are between 0 and " <> show maxIdx
+    GovernanceCmdPollInvalidChoice ->
+      "Invalid choice. Please choose from the available answers."
+    GovernanceCmdDecoderError decoderError ->
+      "Unable to decode metadata: " <> renderDecoderError decoderError
+    GovernanceCmdVerifyPollError pollError ->
+      toS (renderGovernancePollError pollError)
```

Comment:
We need to remove `toS` from the repo (and by extension `cardano-prelude`). Not necessary for this PR.